webapp/src/utils/tooltip_utils.test.ts

```diff
@@ -0,0 +1,30 @@
+import {describe, expect, it} from '@jest/globals';
+
+import {validateGitlabUrl, isValidUrl} from './tooltip_utils';
+
+describe('validateGitlabUrl should work as expected', () => {
+    it('Should return true for valid GitLab repository URL', () => {
+        const URL = 'https://gitlab.com/username/repo/-/merge_requests/1234';
```

Comment:
This is shadowing the global constructor `URL`. Not a big deal here, just thought it was worth mentioning

```suggestion
        const url = 'https://gitlab.com/username/repo/-/merge_requests/1234';
```